pcsx2/GS/GSCapture.cpp

```diff
@@ -536,6 +545,10 @@ bool GSCapture::BeginCapture(float fps, GSVector2i recommendedResolution, float
 			return false;
 		}
 
+		// If the user overrode the pixel format, get that now
+		if (has_pixel_format_override)
```

Comment:
I'm fairly sure this is going to break hardware encoding, since the sw pixel format gets set above. Maybe move this (and the arguments parse) up to before line 458.

Reply:
The arguments parse just converts the string to a dictionary.  The values don't get put into the context until the `avcodec_open2` call.  I don't think there's much we can do about hw encoders if the user tries to override the pixel format, unless we want to parse arguments ourselves.  I'm not too worried about it though, since I think the main reason you'd want to change pixel formats is if you want a lossless codec for something, and don't want chroma subsampling.

Reply:
Right. But check https://github.com/PCSX2/pcsx2/blob/d0ce4c52b0f95c41083537ed13449233e49a6af7/pcsx2/GS/GSCapture.cpp#L489-L490

If the user overrode the pixel format, then the context would be using one format, with the hardware context using another.